projects/angular/src/emphasis/alert/alerts-pager.spec.ts

```diff
@@ -99,6 +100,55 @@ export default function () {
         expect(this.fixture.componentInstance.currentAlert).toEqual(this.alert);
       });
     });
+
+    describe('View Basics', function () {
+      let fixture: ComponentFixture<TestBasics>;
+      let pagerInstance: ClrAlertsPager;
+      let commonStrings: ClrCommonStringsService;
+      const testNextAriaLabel = (current: number, count: number) => {
+        return commonStrings.parse(commonStringsDefault.alertNextAlertAriaLabel, {
+          CURRENT: current.toString(),
+          COUNT: count.toString(),
+        });
+      };
+      const testPreviousAriaLabel = (current: number, count: number) => {
+        return commonStrings.parse(commonStringsDefault.alertPreviousAlertAriaLabel, {
+          CURRENT: current.toString(),
+          COUNT: count.toString(),
+        });
+      };
+
+      beforeEach(() => {
+        const service = new MultiAlertService();
+        const queryList = new QueryList<ClrAlert>();
+        TestBed.configureTestingModule({
+          imports: [ClrEmphasisModule],
+          declarations: [TestBasics],
+          providers: [{ provide: MultiAlertService, useValue: service }],
+        });
+        fixture = TestBed.createComponent(TestBasics);
+        queryList.reset(Array(4).fill(TestBed.createComponent(ClrAlert).componentInstance));
+        service.manage(queryList);
+        fixture.detectChanges();
+        pagerInstance = fixture.componentInstance.pagerInstance;
+        commonStrings = pagerInstance.commonStrings;
+      });
+
+      afterEach(function () {
+        fixture.destroy();
+      });
+
+      it('next alert aria-label is correct after page change', function () {
+        expect(pagerInstance.nextAlertAriaLabel).toBe(testNextAriaLabel(2, 4));
```

Comment:
This is just testing the component properly value, but really we should query the DOM and test the actual `aria-label` attribute.